### PR TITLE
Code Review: Issue 8: Create loan table and add seed data

### PR DESCRIPTION
**Pull request by Nidhi1461996**
#8

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,10 +5,12 @@
   "main": "src/server.js",
   "scripts": {
     "start": "node src/server.js",
-    "test": "jest",
+    "test": "npm run testmigration",
```
Comment (mrsauravsahu) [inline package.json]: You removed `jest` from test in the scripts section. You should run jest when `npm run testmigration` does not fail. Change it to `npm run testmigration && jest`

File: models/loans.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+
+module.exports = (sequelize, DataTypes) => {
+  const loans = sequelize.define('loans', {
+    userID: DataTypes.INTEGER,
```
Comment (mrsauravsahu) [inline models/loans.js]: Should we use `userId` instead of `userID`? We have to use camel case

File: migrations/20180205133032-create-loans.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
+'use strict';
```
Comment (mrsauravsahu) [inline migrations/20180205133032-create-loans.js]: Fix eslint errors.

File: migrations/20180208045308-remove-loanID-column.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+
```
Comment (mrsauravsahu) [inline migrations/20180208045308-remove-loanID-column.js]: Remove empty newlines.

File: models/user.js [hunk 5]
```diff
@@ -1,5 +1,5 @@
 module.exports = (sequelize, DataTypes) => {
-  const User = sequelize.define('User', {
+  const User = sequelize.define('Users', {
```
Comment (mrsauravsahu) [inline models/user.js]: change `Users` to `users`.

File: migrations/20180205133032-create-loans.js [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      userID: {
+        type: Sequelize.INTEGER
+      },
+      loanID: {
+        type: Sequelize.INTEGER
```
Comment (mrsauravsahu) [inline migrations/20180205133032-create-loans.js]: Use camel case. Please replace `loanID` with `loanId`.

File: migrations/20180208045308-remove-loanID-column.js [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    // Remove old attribute
+    queryInterface.removeColumn('loans', 'loanID');
```
Comment (mrsauravsahu) [inline migrations/20180208045308-remove-loanID-column.js]: Camel casing.

File: migrations/20180208045308-remove-loanID-column.js [hunk 8]
```diff
@@ -0,0 +1,12 @@
...
+  },
+
+  down: (queryInterface, Sequelize) => {
+    queryInterface.addColumn('loans', 'loanID', Sequelize.INTEGER);
```
Comment (mrsauravsahu) [inline migrations/20180208045308-remove-loanID-column.js]: Camel casing.

**Review by mrsauravsahu — CHANGES_REQUESTED** [review]

File: models/loans.js (hunk 2 above)
Comment (Ghost---Shadow) [reply models/loans.js]: `userId` is camel case. `userID` is not camel case.

File: models/loans.js [hunk 9]
```diff
@@ -0,0 +1,16 @@
...
+    outstandingAmount: DataTypes.FLOAT,
+    totalAmount: DataTypes.FLOAT,
+    installmentCount: DataTypes.INTEGER,
+  }, {
```
Comment (Ghost---Shadow) [inline models/loans.js]: `loanId` is in the migrations but not in the model.

File: models/user.js (hunk 5 above)
Comment (Ghost---Shadow) [reply models/user.js]: Also rename the file to `users.js`

File: seeders/20180207121247-sample-loans.js [hunk 10]
```diff
@@ -0,0 +1,44 @@
+
+
+module.exports = {
+  up: queryInterface => queryInterface.bulkInsert('loans', [
+    {
+      id: 1,
+      userID: 1,
```
Comment (Ghost---Shadow) [inline seeders/20180207121247-sample-loans.js]: After changing the model and migration to `userId` dont forget to change this too.

File: seeders/20180207121247-sample-loans.js [hunk 11]
```diff
@@ -0,0 +1,44 @@
...
+      totalAmount: 10000,
+      outstandingAmount: 2000,
+      installmentCount: 10,
+      createdAt: new Date(),
+      updatedAt: new Date(),
+    },
```
Comment (Ghost---Shadow) [inline seeders/20180207121247-sample-loans.js]: `loanId` is missing.

**Review by Ghost---Shadow — CHANGES_REQUESTED** [review]
minor changes

File: seeders/20180207121247-sample-loans.js [hunk 12]
```diff
@@ -0,0 +1,44 @@
...
+    {
+      id: 4,
+      userID: 4,
+      totalAmount: 10000,
+      outstandingAmount: 2000,
+      installmentCount: 10,
+      createdAt: new Date(),
+      updatedAt: new Date(),
```
Comment (KshipraShetty) [inline seeders/20180207121247-sample-loans.js:39]: loanId ?

File: seeders/20180207121247-sample-loans.js (hunk 10 above)
Comment (Nidhi1461996) [reply seeders/20180207121247-sample-loans.js]: Yes done

File: seeders/20180207121247-sample-loans.js (hunk 12 above)
Comment (Nidhi1461996) [reply seeders/20180207121247-sample-loans.js:39]: We have removed the loanId column in the latest migration and hence it should not be there while seeding

**Pull request closed without merging**